docs/docs/quickstart/README.md

```diff
@@ -115,7 +115,7 @@ When you **Test** a step, only the current step is executed. Use the caret to te
 
 ## Save data to Google Sheets
 
-Next, create a Google Sheet and add **Timestamp**, **Message** and **Sentiment Score** to the first row. These labels will act as our column headers amd will help us configure the Google Sheets step of the workflow. 
+Next, create a Google Sheet and add **Timestamp**, **Message** and **Sentiment Score** to the first row. These labels act as our column headers amd will help us configure the Google Sheets step of the workflow. 
```

Comment:
```suggestion
Next, create a Google Sheet and add **Timestamp**, **Message**, and **Sentiment Score** to the first row. These labels act as our column headers and will help us configure the Google Sheets step of the workflow.
```

Reply:
Added a comma after 'Message' for better readability (Google's Developer docs style guide), corrected 'amd' to 'and' (typo)